### PR TITLE
test: Fix two failing fun tests

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

**Comment by sschuberth** [comment]
Can I push an update for `OsvServiceFunTest` as well to your branch @fviernau?

**Comment by fviernau** [comment]
> Can I push an update for `OsvServiceFunTest` as well to your branch @fviernau?

I've just read it after filing: https://github.com/oss-review-toolkit/ort/pull/8635.

**Comment by sschuberth** [comment]
> > Can I push an update for `OsvServiceFunTest` as well to your branch @fviernau?
> 
> I've just read it after filing: #8635.

As none of them will separately pass all tests, I'll merge both prematurely.

File: clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt [hunk 1]
```diff
@@ -25,10 +25,12 @@ import io.kotest.matchers.result.shouldBeSuccess
 import io.kotest.matchers.shouldBe
 
 import java.time.Instant
+import kotlinx.serialization.encodeToString
```
Comment (sschuberth) [inline clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt:28]: Left-overs.

**Comment by fviernau** [comment]
> As none of them will separately pass all tests, I'll merge both prematurely.

If meanwhile combined them.

File: clients/osv/src/funTest/assets/vulnerabilities-by-commit-expected-result.json [hunk 2]
```diff
@@ -730,6 +730,7 @@
             "ALSA-2023:4159",
             "ALSA-2023:4175",
             "ALSA-2023:4177",
+            "ALSA-2024:2410",
```
Comment (sschuberth) [inline clients/osv/src/funTest/assets/vulnerabilities-by-commit-expected-result.json:733]: Commit message: s/resuts/results/

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**